esque/cli/commands.py

```diff
@@ -232,7 +233,22 @@ def describe_topic(state, topic_name):
         topic = state.cluster.topic_controller.get_cluster_topic(topic_name)
         config = {"Config": topic.config}
 
+        high_watermark = 0
+        high_watermark_partition_id = 0
+
+        for partition_id in topic.offsets:
+            if topic.offsets[partition_id].high > high_watermark:
+                high_watermark = topic.offsets[partition_id].high
+                high_watermark_partition_id = partition_id
+
+        consumer = MessageConsumer(
```

Comment:
Same holds for the last_message functionality. Maybe something for `topic_controller`?